docs/articles/data_storage.rst

```diff
@@ -256,6 +258,71 @@ Note that above the *force* parameter is necessary if running from a non-interac
 with a migration requires confirmation in the form of user prompt. If running programatically this parameter
 instead confirms your intention to proceed with the migration.
 
+Putting it all together
+  .. code-block::
+
+    import os
+    import synapseutils
+    import synapseclient
+
+    my_synapse_folder_to_migrate = "syn123"
+
+    external_bucket_name = "my-external-synapse-bucket"
+    external_bucket_base_key = "path/within/bucket/"
+
+    # # a path on disk where this utility can create a sqlite database to store its index.
+    # # nothing needs to exist at this path, but it must be a valid path on a volume with sufficient
+    # # disk space to store a meta data listing of all the contents in the indexed entity.
+    # # a rough rule of thumb is 100kB per 1000 entities indexed.
+    db_path = os.path.expanduser("~/synapseMigration/my.db")
+
+    syn = synapseclient.Synapse()
+
+    # # Log-in with ~.synapseConfig `authToken`
+    syn.login()
+
+    # The folder I want to migrate everything to this S3 storage location
+    folder = syn.get(my_synapse_folder_to_migrate)
+
+    folder, storage_location, project_setting = syn.create_s3_storage_location(
```

Comment:
I think we would put the entire project under the storage location. Will project id work?

Reply:
Yes - I just tested this on project: https://www.synapse.org/#!Synapse:syn53017687/files/

And by using the project id it works without issue - I will update the docs accordingly. It is unfortunate that the KW argument in `create_s3_storage_location` is `folder` - But project OR folder works.